src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -34,12 +34,12 @@ public function __construct($namespace = '', $defaultLifetime = 0, $nonce = null
         }
         parent::__construct($namespace, $defaultLifetime);
 
-        if (null !== $nonce) {
-            CacheItem::validateKey($nonce);
+        if (null !== $version) {
+            CacheItem::validateKey($version);
 
-            if (!apcu_exists($nonce.':nonce'.$namespace)) {
+            if (!apcu_exists($version.':version'.$namespace)) {
                 $this->clear($namespace);
-                apcu_add($nonce.':nonce'.$namespace, null);
+                apcu_add($version.':version'.$namespace, null);
```

Comment:
If `$namespace = 'my_namespace'` and `$version = 'v3'`, the we'll use `v3:versionmy_namespace`. Are we sure we don't want to do something like: `v3:my_namespace` or `v3:version:my_namespace` ?